lib/modules/apostrophe-login/index.js

```diff
@@ -160,25 +160,26 @@ module.exports = {
     // If the user's login SUCCEEDS, the return value is
     // the `user` object.
 
-    self.verifyLogin = async function(username, password) {
+    self.verifyLogin = async function(username, password, done) {
       const req = self.apos.tasks.getReq();
       const user = await self.apos.users.find(req, {
         $or: [
           { username: username },
           { email: username }
         ],
         disabled: { $ne: true }
-      });
+      }).toObject();
+
       if (!user) {
         await Promise.delay(1000);
-        throw 'invalid';
+        throw 'No user found.';
```

Comment:
Please leave this as `invalid`, string exceptions are used to communicate specific meaningful error codes to the browser side.